### PR TITLE
chore(es6): add es6 support

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: .eslintrc.es6.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "extends": "ovh",
+    "globals": {
+        "window": false,
```
Comment (bnjjj) [inline .eslintrc.es6.json]: It seems there is a mistake anywhere because you don't have to do that I think it's already included in ovh-config

Comment (Jisay) [inline .eslintrc.es6.json]: I will it try but it was already into the eslintrc.json files before I deleted it

File: Gruntfile.js [hunk 2]
```diff
@@ -22,6 +22,23 @@ module.exports = function (grunt) {
...
     var mode = grunt.option("mode") || "dev";
 
+    var copyFiles = [
+        "*.{ico,png,txt}",
+        "bower_components/**/*.{ttf,woff,woff2,svg,eot}",
+        "bower_components/bootstrap/dist/css/bootstrap.min.css",
+        "bower_components/angular-i18n/**.js",
+        "components/**/*.html",
+        "components/**/assets/**/*",
+        ".htaccess",
```
Comment (antleblanc) [inline Gruntfile.js]: @lizardK @frisouilles @blaryjp Is there any reason to copy `.htaccess` file? This one does not exist in our this repository.

File: Gruntfile.js [hunk 3]
```diff
@@ -389,58 +454,42 @@ module.exports = function (grunt) {
...
+                }, {
+                    expand : true,
+                    cwd: "<%= yeoman.tmp %>",
+                    dest: "<%= yeoman.dist %>/client",
+                    src : ["**/translations/*.json"]
+                }, {
+                    cwd: "<%= yeoman.client %>",
+                    expand: true,
+                    dest: "<%= yeoman.dist %>/client",
+                    src: "usertests/**/*"
```
Comment (antleblanc) [inline Gruntfile.js]: Maybe we can remove this object right? Because doesn't exist any folders `usertests`.

**Pull request closed without merging**